cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/utils/SerialisationUtil.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest.utils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class SerialisationUtil {
+
+  private SerialisationUtil() {}
+
+  public static Map<String, String> stringToMap(String serialisedMap) {
+    return stringToMap(serialisedMap, ",", "=");
+  }
+
+  public static Map<String, String> stringToMap(String serialisedMap, String entrySeparator,
+      String keyValueSeparator) {
+    if (serialisedMap == null || serialisedMap.trim().isEmpty()) {
+      return new LinkedHashMap<>();
+    }
+    return Arrays.stream(serialisedMap.split(entrySeparator))
+        .map(s -> s.split(keyValueSeparator))
+        .collect(Collectors.toMap(s -> s[0], s -> s[1]));
```

Comment:
Feels like this could potentially fail with an` IndexOutOfBounds` exception on the `[1]` if the caller gives a dodgy `serialisedMap` param that doesn't contain an `entrySeparator` within it.
Do we need some sort of checking first?

Reply:
Ah ok, that's fine sounds, it should be safe enough as is then. Not necessary.